toolkit/components/uniffi-bindgen-gecko-js/src/templates/js/Error.jsm

```diff
@@ -0,0 +1,51 @@
+{%- let error = ci.get_error_definition(name).unwrap() %}
+{%- let string_type = Type::String %}
+{%- let string_ffi_converter = string_type.ffi_converter() %}
+
+class {{ error.nm() }} extends Error {}
+EXPORTED_SYMBOLS.push("{{ error.nm() }}");
+
+{% for variant in error.variants() %}
+class {{ variant.name().to_camel_case() }} extends {{ error.nm() }} {
```

Comment:
would have been nice to create a `variant.nm()` ext, but `Variant` isn't exposed from `uniffi_bindgen`.

Created to https://github.com/mozilla/uniffi-rs/pull/1202 to fix that 😄 (not needed, just keeps things consistent)